docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,221 @@
+-   In the **Solution Explorer** view, right-click the solution name and select **Build**.
+
+Tizen .NET applications are always deployed as installable packages. The package files have the `.tpk` file extension, and the process of generating a package is controlled by the [manifest file](../../../vstools/tools/manifest-editor.md). The Visual Studio template generates the manifest file (`tizen-manifest.xml`) to the top level of the **\<projectname\>.Tizen project**.
+
+For this example application, the default manifest is sufficient. If you want to make any change in the application, such as changing the application icon or installing resources that are used by the application at runtime, see [Package Your Application](#package-your-application).
+
+After you build the application, deploy and run it.
+
+## Deploy and Run Your Application
+
+To run the UI application, you must first deploy it to the reference board with the headed image installed. Deploying means transferring the package file (`.tpk`) to the target and invoking the Tizen Package Manager to install it.
+
+To deploy and run the application on the reference board:
+
+1. Connect the reference board to your PC.
+
+2. Once you connect the reference board, you can deploy the application by clicking the reference board instance in the Visual Studio toolbar. Make the Tizen project as a start-up project with **Set as StartUp Project**.
+
+    ![Deploy your package](media/vs_device_launch.png)
+
+    In the Visual Studio toolbar, you can select the reference board from the drop-down list to change the deployment target.
+
+
+3. If the deployment is successful, the application icon is visible on the device screen. Click the icon to launch the application.
+
+    The following figure shows the launched application on the reference board:
+
+    ![Application running on the reference board](media/cs_first_building_device.png)
+
+Visual Studio uses the Smart Development Bridge (SDB) to communicate with the reference board. If you encounter a problem with detecting the device in Visual Studio, you can check the SDB manually:
+
+1.  In the Visual Studio menu, select **Tools \> Tizen \> Tizen Sdb Command Prompt**.
+2.  In the command prompt, enter `sdb devices`.
+
+    ![Device detection](media/cs_first_building_sdb_prompt.png)
+
+    A list of attached devices appears.
+
+If you face any issue during deployment, it is recommended to manually install the application using SDB:
+
+-   IoT application:
+
+    ```bash
+    $ sdb install <path-to-package>/org.tizen.example.CrossTemplate1.Tizen-1.0.0.tpk
+    ```
+
+## Understand Your Application
+
+The C\# code from your first application displays a label centered on the screen, containing the **Welcome to Xamarin Forms!** text. This CrossTemplate1 application created from the template is set up and ready to be built and run by Visual Studio right after you create it, as described above.
```

Comment:
@mijin85cho Don't we need to maintain consistency here and change all over other files too like in https://docs.tizen.org/application/dotnet/get-started/mobile/first-app/?
I agree with your comment and it would be actually good.

Reply:
@nawab-samsung Check the accuracy of the content.
In the link you marked, the actual string printed on the screen is "Welcome to Xamarin Forms!"
![image](https://user-images.githubusercontent.com/9958374/83718046-2e80e000-a66f-11ea-8577-5cd700931223.png)

![image](https://user-images.githubusercontent.com/9958374/83718240-9e8f6600-a66f-11ea-8b14-63261699df8d.png)


In this page the string that coded and printed is "Welcome to Xamarin.Forms!"

![image](https://user-images.githubusercontent.com/9958374/83718124-66882300-a66f-11ea-8cf3-b0ae69ef41e6.png)

![image](https://user-images.githubusercontent.com/9958374/83718161-7bfd4d00-a66f-11ea-8f34-a1ad6f5f3511.png)

Please read and understand the content, to get what you should check out...

Reply:
@mijin85cho Thanks but I think you did not get my point here: 
1. The text above or below the image is not pointing to "welcome to Xamarin forms" its in code segment that "Welcome to Xamarin.Forms" exists. We usually do not alter code. 

2. I was originally pointing out to line 130 which is exactly written as per other existing guides, have a look please:  

![11](https://user-images.githubusercontent.com/43839737/83719527-e5bc2d80-a654-11ea-94c3-aaf03200339f.PNG)

I don't think I am wrong in my observation here, but you took my comment wrongly here.
Kindly verify and in case of any query or discrepancy, let me know. I will be glad to clarify

Reply:
@mijin85cho can we match the image text to "Welcome to Xamarin Form", then all will be aligned to match other guides. What is your opinion on this?

Reply:
@nawab-samsung Now I see that I misread your previous comment and I'm sorry for that.

"Matching the string to the actual code" & " Updating the image" both are possible way to get the accuracy of the content. 
For the case of image updating, @xbldx should rebuild the app with updated "Text" and get the image of the result. I think we can accept any way that @xbldx choose if the strings are matched.

Reply:
![334](https://user-images.githubusercontent.com/43839737/83721114-10f44c00-a658-11ea-8aba-ee6ca1321645.PNG)

Reply:
I fixed the image.